manage_breast_screening/clinics/jinja2/clinics/show.jinja

```diff
@@ -65,6 +64,31 @@
         {{ tag(presented_appointment.special_appointment_tag_properties) }}
       {% endif %}
     {% endset %}
+    {% set appointment_html %}
+      {{
+        appointment_status(
+          appointment=presented_appointment,
+          check_in_url=url(
+            'clinics:check_in',
+             kwargs={
+              'pk': presented_clinic.pk,
+              'appointment_pk': presented_appointment.pk,
+              }
+          ),
+          csrf_input=csrf_input
+        )
+      }}
+      <p class="nhsuk-u-margin-top-2 nhsuk-u-margin-bottom-0 app-nowrap">
```

Comment:
Does the `.app-nowrap` exist in Sass?

It's [from the prototype](https://github.com/NHSDigital/manage-breast-screening-prototype/blob/d5d8540556c585ee66d929b230a6a800ba8dec14/app/assets/sass/_utils.scss#L4)

~Do we definitely want something unpredictable in length (like a full name) to never wrap?~

Ah it's visually hidden so all good, but do check the class exists